runit.fish

```diff
@@ -0,0 +1,2 @@
+node run.js ../heroku-apps ../heroku-certs ../heroku-ci ../heroku-cli-addons ../heroku-cli-oauth ../heroku-container-registry ../heroku-fork ../heroku-git ../heroku-local ../heroku-orgs ../heroku-pg ../heroku-pipelines ../heroku-ps-exec ../heroku-run ../heroku-spaces ../heroku-status
```

Comment:
I think it would be better to add all these to the package.json and require them directly in the code.